_posts/2020-11-21-debenedetto20a.md

```diff
@@ -25,10 +25,10 @@ lastpage: 2420
 page: 2412-2420
 order: 2412
 cycles: false
-bibtex_author: Debenedetto, Justin and Chiang, David
+bibtex_author: DeBenedetto, Justin and Chiang, David
```

Comment:
```suggestion
bibtex_author: De{B}enedetto, Justin and Chiang, David
```

Reply:
Hi Justin, would this be better? I'm not sure what Bibtex does with letters inside the name. But it feels more secure to force it to capitalize in the reference (the bibtex field is drawn from here).